packages/itwinui-react/src/core/Slider/Slider.tsx

```diff
@@ -152,6 +152,26 @@ export type SliderProps = {
    * Additional props for container `<div>` that hold the slider rail, thumbs, and tracks.
    */
   railContainerProps?: React.HTMLAttributes<HTMLDivElement>;
+  /**
+   * Allows props to be passed for slider-min
+   */
+  sliderMinProps?: React.ComponentProps<'span'>;
+  /**
+   * Allows props to be passed for slider-max
+   */
+  sliderMaxProps?: React.ComponentProps<'span'>;
+  /**
+   * Allows props to be passed for slider-container
+   */
+  sliderContainerProps?: React.ComponentProps<'div'>;
```

Comment:
there is already `railContainerProps`, so `sliderContainerProps` is not needed

Reply:
fixed

Reply:
you still need to fix the existing `railContainerProps` to actually merge the classes correctly

Reply:
fixed